src/pages/ResourceView.tsx

```diff
@@ -23,6 +47,48 @@ export default function ResourceView() {
         </a>
       </div>
       <ResourceCard resource={resource} onUpdate={onUpdateResource} />
+      <hr style={{ borderTopWidth: '3px' }} />
```

Comment:
If this is copied from elsewhere (it looks familiar), it'd be good to extract a `<Divider />` component for shared usage.

Reply:
good idea 👍

Reply:
would it make sense to extract the back button into a component as well? Seems like its used in a few places

Reply:
Good catch. There's a back button in the PageHeader component which this page should use, just make sure to pass the `showBack` prop to it.